.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 19.10b0
+    rev: 21.4b0
```

Comment:
this is the repo's file. I guess you want to change this instead https://github.com/OCA/oca-addons-repo-template/blob/master/src/.pre-commit-config.yaml.jinja#L4 ?

Reply:
Ok, thanks for the advice @simahawk!